### ISSUE TITLE
Exit code 1 using process substitutions

### ISSUE THREAD

**Issue by jeissonh** [jeffkaufman/icdiff#205]
[Process substitution](https://en.wikipedia.org/wiki/Process_substitution) combined with diff is a very practical way to compare direct output of programs. However from version 2.0.4 icdiff finishes with exit code 1 when process substitution is used although content is identical. E.g:

```sh
$ echo 'identical content' > file.txt

$ icdiff file.txt file.txt
$ echo $?
0

$ icdiff file.txt <(echo 'identical content')
file.txt                               /dev/fd/63         
$ echo $?
1

$ icdiff --no-headers file.txt <(echo 'identical content')
$ echo $?
1
```

Exit code 1 makes very difficult makefile automation. E.g:

```sh
icdiff --no-headers output001.txt <(program < input001.txt)
make: [output001.txt] Error 1 (ignored)
icdiff --no-headers output002.txt <(program < input002.txt)
make: [output002.txt] Error 1 (ignored)
icdiff --no-headers output003.txt <(program < input003.txt)
make: [output003.txt] Error 1 (ignored)
...
```

I know make's Error 1 messages can be suppressed with syntax `-icdiff file1 file2 ||:`. However this may silence other important error messages.

Thanks a lot in advance!

**Comment by jeffkaufman** [comment]
I think 9526c21 should fix it; let me know if it doesn't?

**Comment by jeissonh** [comment]
In fact the commit solves it. Thank you so much! I hope it get released soon.

**Comment by jeffkaufman** [comment]
Released in 2.0.6

**Comment by jeissonh** [comment]
Wonderful! I am very thankful for your awesome work!
